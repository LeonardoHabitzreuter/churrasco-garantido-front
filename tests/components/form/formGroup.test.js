import React from 'react'
import { shallow } from 'enzyme'
import FormGroup from '../../../src/components/form/formGroup'

describe('form component', () => {
  test('should render properly', () => {
    const wrapper = shallow(<FormGroup />)
    expect(wrapper).toMatchSnapshot()
  })

  test('should has the className when is passed by the props', () => {
    const wrapper = shallow(<FormGroup className='Test' />)
    expect(wrapper.find('.Test').exists()).toBeTruthy()
  })
})
