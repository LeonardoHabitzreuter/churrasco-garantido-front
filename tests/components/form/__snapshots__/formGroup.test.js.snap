// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form component should render properly 1`] = `
<FormGroup
  bsClass="form-group"
  className="col-sm-12"
>
  <Col
    bsClass="col"
    componentClass={[Function]}
    sm={2}
  />
  <Col
    bsClass="col"
    componentClass="div"
    sm={4}
  >
    <FormControl
      bsClass="form-control"
      componentClass="input"
    />
    <FormControlFeedback
      bsClass="form-control-feedback"
      bsRole="feedback"
    />
  </Col>
</FormGroup>
`;
